client/src/components/sidebar/friends/RemoveFriend.tsx

```diff
@@ -0,0 +1,146 @@
+import { Alert, Button, Dialog, Popover, Snackbar, Typography } from '@mui/material';
+import { styled } from '@mui/system';
+import { useState } from 'react';
+
+export const emptyProfile = 'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png';
+
+const RemoveFriendDialog = styled(Dialog)(({ theme }) => ({
+  '& .MuiDialog-paper': {
+    padding: theme.spacing(1),
+    borderRadius: theme.shape.borderRadius,
+    minWidth: '400px',
+    minHeight: '250px',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+}));
+
+const StyledContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  margin: 25px;
+  gap: 15px;
+`;
+
+const StyledButtonContainer = styled('div')`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  gap: 20px;
+  justify-content: center;
+  align-items: center;
+`;
+
+const CancelRemove = styled(Button)(({ theme }) => ({
+  backgroundColor: theme.palette.grey[300],
+  color: theme.palette.text.primary,
+  width: '50%',
+  height: '40px',
+  textTransform: 'none',
+  fontWeight: 500,
+  '&:hover': {
+    backgroundColor: theme.palette.grey[400],
+  },
+}));
+
+const ConfirmRemove = styled(Button)(({ theme }) => ({
+  backgroundColor: theme.palette.error.light,
+  color: theme.palette.error.contrastText,
+  width: '50%',
+  height: '40px',
+  textTransform: 'none',
+  fontWeight: 500,
+  '&:hover': {
+    backgroundColor: theme.palette.error.main,
+  },
+}));
```

Comment:
remove the height from the buttons - they're fine as they are

```suggestion
const CancelRemove = styled(Button)(({ theme }) => ({
  backgroundColor: theme.palette.grey[300],
  color: theme.palette.text.primary,
  width: '50%',
  textTransform: 'none',
  fontWeight: 500,
  '&:hover': {
    backgroundColor: theme.palette.grey[400],
  },
}));

const ConfirmRemove = styled(Button)(({ theme }) => ({
  backgroundColor: theme.palette.error.light,
  color: theme.palette.error.contrastText,
  width: '50%',
  textTransform: 'none',
  fontWeight: 500,
  '&:hover': {
    backgroundColor: theme.palette.error.main,
  },
}));
```